lib/session_setup/view/session_setup_page.dart

```diff
@@ -56,34 +56,44 @@ class SessionSetupView extends StatelessWidget {
               : null,
           body: Padding(
             padding: const EdgeInsets.all(8.0),
-            child: Column(
-              children: [
-                Text(
-                  'Choose a prompt bellow:',
-                  style: Theme.of(context).textTheme.headline3,
-                ),
-                TextField(
-                  onChanged: context.read<SessionSetupCubit>().changedQuery,
-                ),
-                Expanded(
-                  child: SingleChildScrollView(
-                    child: Wrap(
-                      children: [
-                        for (final prompt in state.promptList)
-                          PromptCard(
-                            prompt: prompt,
-                            selected: prompt == state.selectedPrompt,
-                            onTap: () {
-                              context
-                                  .read<SessionSetupCubit>()
-                                  .selectPrompt(prompt);
-                            },
-                          ),
-                      ],
+            child: Center(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  Text(
+                    'Choose a prompt bellow:',
+                    style: Theme.of(context).textTheme.headline1,
+                  ),
+                  SizedBox(
```

Comment:
A padded text field with a description on what the prompt is for might be nice, but optional 😄

Reply:
Not sure I understood, gonna merge as it is because of the other improvements and afterwards we can do this

Reply:
Yep, agreed! 💪🏻